src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -125,7 +125,9 @@ class IndexLogEntryTest extends SparkFunSuite with SQLHelper with BeforeAndAfter
         |                  "kind" : "NoOp",
         |                  "properties" : { }
         |                }
-        |              }
+        |              },
+        |              "excluded" : ["file:/rootpath/f1"],
+        |              "appended" : ["file:/rootpath/f3"]
```

Comment:
nit: PR description talks about `excluded` but looks like you're handling `appended` as well? Can you update the text throughout for consistency?

Reply:
PR description is updated.